### ISSUE TITLE
[Quest] [Hillsbrad Foothills] Dangerous! unable to pickup

### ISSUE THREAD

**Issue by BigDawgBen** [dalaranwow/dalaran-wow#5665]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Quest showing up on minimap but unavailable to pickup from questgiver - no exclamation mark, either yellow or greyed.

**Expected behaviour**: Tell us what should happen instead
At level 28 it should be avail for pickup, as indicated on minimap.

**Steps to reproduce the problem**:

1. Go to Tarren Mill.
2. Pick up  all the quests.
3. Left with one that can't be picked up anywhere.

**Include proofs for this behaviour**
Just trying to help.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://www.wowhead.com/quest=567/deprecated-dangerous

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Golganeth** [comment]
Works as intended, make sure you meet the requirements and that you have checked carefully.
Closing

**Comment by BigDawgBen** [comment]
Working now, not sure what the issue was before, thanks for clearing up.
